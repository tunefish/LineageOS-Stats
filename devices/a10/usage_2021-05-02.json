{
  "10.3": 1,
  "16.0": 78,
  "17.1": 979,
  "18.1": 593,
  "none": 553
}