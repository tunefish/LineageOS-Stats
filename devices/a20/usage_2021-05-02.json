{
  "17.1": 2435,
  "18.1": 301,
  "none": 28
}