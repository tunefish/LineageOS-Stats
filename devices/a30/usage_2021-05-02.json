{
  "17.1": 39,
  "18.0": 787,
  "18.1": 516,
  "none": 4
}