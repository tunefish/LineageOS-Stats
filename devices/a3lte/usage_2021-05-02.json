{
  "15.1": 772,
  "16.0": 287,
  "17.1": 1008
}