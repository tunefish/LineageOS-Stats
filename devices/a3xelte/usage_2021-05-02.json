{
  "14.1": 291,
  "15.1": 151,
  "16.0": 561,
  "17.1": 5124,
  "18.0": 3,
  "18.1": 24,
  "52.8": 1,
  "none": 35
}