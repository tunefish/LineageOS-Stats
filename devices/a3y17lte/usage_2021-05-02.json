{
  "14.1": 5585,
  "15.1": 451,
  "16.0": 1291,
  "17.1": 1583,
  "18.0": 1,
  "18.1": 2
}