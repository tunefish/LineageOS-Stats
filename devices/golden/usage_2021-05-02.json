{
  "13.0": 1369,
  "14.1": 4862,
  "15.1": 6,
  "17.1": 12,
  "none": 1
}